org-formation/800-redirects/_tasks.yaml

```diff
@@ -97,6 +97,23 @@ DcaStagingAppDnsForward:
 
 # forward dca.app.sagebionetworks.org to data_curator-infra ALB
 # https://github.com/Sage-Bionetworks/data_curator-infra
+DcaTestingAppDnsForward:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.6.9/templates/R53/cname.yaml
+  StackName: !Sub '${resourcePrefix}-dca-testing-cname'
+  StackDescription: Setup a CNAME for data_curator-infra testing ALB
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    Account: !Ref SageITAccount
+  Parameters:
+    # the name of the CNAME record
+    SourceHostName: "dca-testing.app.sagebionetworks.org"
+    # ID of the app.sagebionetworks.org zone (in sageit account)
+    SourceHostedZoneId: !CopyValue [!Sub '${primaryRegion}-${resourcePrefix}-sagebio-app-zone-HostedZoneId']
+    # the value of the CNAME record
+    TargetHostName: !CopyValue ['dca-testing-DockerFargateStack-LoadBalancerDNS', !Ref DCAProdAccount]
```

Comment:
I checked to make sure that this stack output exists in AWS (because this deploy pipeline will fail if it doesn't), and I don't see a stack for dca-testing in CloudFormation yet.

I think whats missing is a testing branch and an update [here](https://github.com/Sage-Bionetworks/data_curator-infra/blob/dev/.github/workflows/aws-deploy.yml) to deploy the testing environment from that branch. 

@afwillia can you confirm that's how the testing environment should be deployed?

Reply:
@jesusaurus yeah, that's right. I forgot that data_curator-infra needs a testing branch to deploy this. Thank you!

I'll ping you for re-review when the stack is available on CloudFormation.

Reply:
@jesusaurus this should be ready for re-review now. The dca-testing stack is in cloudFormation